executor/memtable_reader_test.go

```diff
@@ -1068,6 +1071,21 @@ func (s *testHotRegionsHistoryTableSuite) TearDownSuite(c *C) {
 	}
 	s.testInfoschemaTableSuiteBase.TearDownSuite(c)
 }
+func bytesKeyToHex(key []byte) string {
```

Comment:
We already have this function in `helper.go`

Reply:
I add an exported `BytesKeyToHex`.